control-plane/api-gateway/binding/result.go

```diff
@@ -36,6 +36,7 @@ var (
 	errRouteNoMatchingParent                = errors.New("no matching parent")
 	errInvalidExternalRefType               = errors.New("invalid externalref filter kind")
 	errExternalRefNotFound                  = errors.New("ref not found")
+	errFilterInvalid         = errors.New("filter invalid")
```

Comment:
gofmt

Reply:
I really gotta figure out why my editor has been doing this format after save recently rather than before save